core/primitives/scheduled_change.cpp

```diff
@@ -0,0 +1,15 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "primitives/scheduled_change.hpp"
+
+#include "primitives/common.hpp"
+#include "scale/collection.hpp"
+#include "scale/fixedwidth.hpp"
+
+// it's going to be necessary later when refactoring scale
```

Comment:
At least comment out the includes except the first one in order to make compilation (I think) faster

Reply:
done